windows_containers/wmco_rn/windows-containers-release-notes-10-17-x-past.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+:context: windows-containers-release-notes-10-17-x-past
+[id="windows-containers-release-notes-10-17-x-past"]
+= Release notes for past releases of the Windows Machine Config Operator
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+The following release notes are for previous versions of the Windows Machine Config Operator (WMCO).
+
+For the current version, see xref:../../windows_containers/wmco_rn/windows-containers-release-notes-10-17-x.adoc#windows-containers-release-notes-10-17-x[{productwinc} release notes].
+
+[id="wmco-10-17-"]
+== Release notes for Red Hat Windows Machine Config Operator 10.17.0
+
+This release of the WMCO provides bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 10.17.0 were released in
+
+
+[id="wmco-10-17-past-new-features"]
+=== New features and improvements
```

Comment:
This PR is intended to add recent changes I am making to the WMCO release notes branch-by-branch to `main` so that future version will get the proper organization. It create a template or placeholder for future releases, so that the deltas will be fewer (I hope).

Reply:
@kalexand-rh Do you see an issue with making these changes to `main` and leaving "blank" WMCO release notes in the branch. I think the only folks who will see the blanks would be users of the OKD latest docs. Currently, they are seeing [WMCO 6.0.0 release notes](https://docs.okd.io/latest/windows_containers/windows-containers-release-notes-6-x.html).

Reply:
I think it's ok to temporarily leave them blank.